lib/organisms/widgets-area/widgets-area.directive.js

```diff
@@ -0,0 +1,30 @@
+angular
+  .module('lnPatterns')
+  .directive('lnOWidgetsArea', lnOWidgetsArea);
+
+lnOWidgetsArea.$inject = ['$compile'];
+
+function lnOWidgetsArea($compile) {
+  return {
+    restrict: 'A',
+    link: link,
+    scope: {
+      lnWidgets: '<'
+    }
+  };
+
+  function link(scope, elem) {
+    if (!scope.lnWidgets) {
+      return;
+    }
+
+    elem.empty();
+
+    angular.forEach(scope.lnWidgets, function(widget, index) {
+      var widgetDirective = '<section class="' + widget.class + '" ' + widget.type + ' ln-content="lnWidgets[' + index + '].content"></section>';
```

Comment:
I think we should generate the class name automatically from the type, because I don't think we would pass this field in the api.